### PR TITLE
MSPSDS-856: Tidy docker setup

### PR DESCRIPTION
**Pull request by broder**
This will be quite a breaking change for people's local setups...

It'll probably be easier to review this commit by commit.

### CODE REVIEW

File: cosmetics-web/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Cosmetics Website
+
+This folder contains the code for the Cosmetics website.
+This folder also contains the code for the (background worker)[../cosmetics-worker/README.md].
```
Comment (jasiekmiko) [inline cosmetics-web/README.md]: I think you've got the parentheses and brackets the wrong way 'round here?

File: cosmetics-worker/README.md [hunk 2]
```diff
@@ -33,13 +33,6 @@ This provisions the app in Cloud Foundry.
 
 This configures rails to use the production database amongst other things.
 
-    cf set-env cosmetics-worker AWS_ACCESS_KEY_ID XXX
```
Comment (jasiekmiko) [inline cosmetics-worker/README.md]: Are these simply not used by cosmetics at this point in time? (it's only for S3 that we use it on mspsds, isn't it?)

Comment (broder) [reply cosmetics-worker/README.md]: S3 is not used yet. It should be fairly straightforward to add these lines back in when it does start using it.

File: cosmetics-web/README.md (hunk 1 above)
Comment (broder) [reply cosmetics-web/README.md]: Good spot!

**Pull request merged**